otherlibs/dune-rpc-lwt/src/dune_rpc_lwt.ml

```diff
@@ -84,8 +84,12 @@ module V1 = struct
     Where.Make
       (Fiber)
       (struct
+        let lwt_result_catch f =
```

Comment:
Since this function is only used once, what about just inlining it?

Reply:
https://github.com/ocaml/dune/pull/6583/commits/b624cf3699b0b2010251d7a8f0775a661663d2f5